.changeset/loud-swans-serve.md

```diff
@@ -0,0 +1,5 @@
+---
+"@envelop/opentelemetry": minor
```

Comment:
Linting is failing on this file. Another change set which passed looks like:
```
---
'@envelop/validation-cache': patch
---

fix: validation cache for ESM environments
```

So my guess is its failing on the double quotes and is expecting single quotes. Probably best to also update the message to be conventional.

Reply:
Hi @shift thanks I have updated this now.